locales/en.json

```diff
@@ -43,26 +43,26 @@
     "flipCamera": "Flip Camera"
   },
   "VcDetails": {
-    "generatedOn": "Generated on",
+    "generatedOn": "Generated On",
     "status": "Status",
     "valid": "Valid",
     "photo": "Photo",
-    "fullName": "Full name",
+    "fullName": "Full Name",
     "gender": "Gender",
-    "dateOfBirth": "Date of birth",
-    "phoneNumber": "Phone number",
+    "dateOfBirth": "Date of Birth",
+    "phoneNumber": "Phone Number",
     "email": "Email",
     "address": "Address",
     "reasonForSharing": "Reason for sharing",
-    "idType": "ID type",
+    "idType": "ID Type",
     "id": "Id",
     "qrCodeHeader": "QR Code",
     "nationalCard": "National Card",
     "uin": "UIN",
     "vid": "VID",
     "enableVerification": "Activate",
-    "profileAuthenticated": "Activated for online login!",
-    "offlineAuthDisabledHeader": "Activation pending for online login!",
+    "profileAuthenticated": "Credentials are enabled for online authentication.",
```

Comment:
Is this going to be changed for other locales?

Reply:
No need for change in others. Just check with Filipinos, since most of the words were similar to English in that language.

Reply:
checked in filipino. removed '!'